src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -590,11 +592,10 @@ private void transferAlignmentInfoToPairedRead(final SAMRecord firstUnaligned, f
      * Checks to see whether the ends of the reads overlap and soft clips reads
      * them if necessary.
      */
-    protected void clipForOverlappingReads(final SAMRecord read1, final SAMRecord read2) {
+    protected static void clipForOverlappingReads(final SAMRecord read1, final SAMRecord read2) {
```

Comment:
No, it needs to be protected or package level for the tests to be able to access it.